README.md

```diff
@@ -109,22 +109,48 @@ direct you to a page that gives you the ability to submit a request to the
 master repository to merge in the changes you committed.
 
 ##Style Guide
+### Expected Values
+
+When a value is expected or described within plain text, but isn't specifically code, single quotes should be used.
+In situations where it is important to group text to be specific to the concept, single quotes should also be used.
+Single quotes are also allowable if the text would be unclear due to certain property names.
+
+For example:
+
+The reserved Verb 'http://adlnet.gov/expapi/verbs/voided' is an exception.
```

Comment:
If you indent the examples they will have the same appearance as the once below.